slides.Rmd

```diff
@@ -54,53 +56,107 @@ table_contents2 <- data.frame(c("&", "|",
 
 - R is open source and free to use, and available for all major operating systems
 
+- It is part of the analytical strategy but R is not the only tool we can use
 
-## R or RStudio?
 
-RStudio is a free and open-source integrated development environment for R, a programming language for statistical computing and graphics. You cannot run RStudio without R.
+## Difference between R and RStudio?
 
+- RStudio is a free and open-source integrated development environment for R, a programming language for statistical computing and graphics. You cannot run RStudio without R.
 - RStudio is more flexible and powerful, and provides direct access to R code
 - Both are good for viewing data, neither is good for manually editing data
 - Use RStudio for any project that requires direct interaction with code and/or manipulation of complex data
+- RStudio Pro is proprietary and comes with user support from RStudio and some extra functionality like the use of RStudio server.
 
-## Functions and data embedded in R
 
-```{r,  eval=FALSE, echo=TRUE}
-summary(cars)
+## RStudio
+![RStudio](images/RStudio_Screenshot.png){ width=100%, height=100%}
+```{r, echo = FALSE}
+# TRAINERS NOTES
+# Explain what each window does
+# Note that the console is for instantly executing a line of code and is different from the script
 ```
 
-```{r}
-summary(cars)
+
+## R packages 
+
+- R packages are collections of functions and, sometimes, datasets created by the R community. 
+
+- R packages cover a wide range of needs and can improve the basic functionality of R or add additional functionality. 
+
+- Packages come with extensive helpfiles called vignettes. These contain information on all the functions in the package, a description of what the packaage is designed to do
+
+- Link to package vignette - suggestions of a good one?
```

Comment:
Maybe [Lubridate](https://cran.r-project.org/web/packages/lubridate/vignettes/lubridate.html), it's hopefully not _too_ complicated but it is something useful which we don't cover in theses sessions?

Reply:
Thanks! That one looks good.